licenses.d2iq.yaml

```diff
@@ -630,3 +631,13 @@ resources:
       - url: https://github.com/kubernetes/kubernetes
         ref: master
         license_path: LICENSE
+  - container_image: docker.io/bitnami/kubectl:1.30.5
+    sources:
+      - url: https://github.com/kubernetes/kubectl
+        ref: v0${image_tag#1}
+        license_path: LICENSE
+  - container_image: docker.io/bitnami/postgresql:15.8.0-debian-12-r14
```

Comment:
There's a patched d2iq container found [here](https://github.com/mesosphere/kommander-applications/blob/012ea26aecbaf19141e210b0f4c4b42aa6db06e9/licenses.d2iq.yaml#L608) from v2.12.0 : `ghcr.io/mesosphere/dkp-container-images/docker.io/bitnami/postgresql:15.2.0-debian-11-r21-d2iq.0` use this one instead?